src/modal/modal.scss

```
@@ -66,6 +69,11 @@
         margin-right: $iui-s;
       }
     }
+
+    .iui-modal-content {
+      flex-grow: 2;
+      overflow: scroll;
```

```suggestion
      overflow-y: auto;
      overflow-y: overlay;
```
I would suggest targeting only overflow on the Y axis.  Also I'd like to use overlay with a fallback for browsers that don't support it.

---

Done